package.json

```diff
@@ -27,24 +28,19 @@
   "peerDependencies": {
     "eslint-plugin-mocha": "^2.0.0",
     "eslint-plugin-react": "^4.1.0",
-    "eslint-plugin-json": "^1.2.0",
```

Comment:
I was following the pattern established in [`airbnb/javascript` for `peerDependencies`]. 

_... puts on thinking cap ..._ 

If we want to be backwards compatible for both `npm@2` *AND* `npm@3` then I think we need to keep these in `peerDependencies` (but obviously make them more liberal – like `"*"`). If we want to _only_ support `npm@3` then we can keep them as `dependencies` since they will be flattened by default.

Another option (which I have NO idea will work) is to use the `require.resolve` pattern used for `"extends"` for `"plugins"`. i.e. change:

``` js
{
  plugins: ['mocha', 'json'],
}
```

to use `require.resolve`:

``` js
{
  plugins: ['eslint-plugin-mocha', 'eslint-plugin-json']
    .map(require.resolve),
}
```

[`airbnb/javascript` for `peerDependencies`]: https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1